static/js/recent_topics.js

```diff
@@ -317,6 +319,18 @@ function format_topic(topic_data) {
     const senders = all_senders.slice(-MAX_AVATAR);
     const senders_info = people.sender_info_for_recent_topics_row(senders);
 
+    const extra_sender_ids = all_senders.slice(0, -MAX_AVATAR);
+    const displayed_other_senders = extra_sender_ids.slice(-MAX_EXTRA_SENDERS);
+    let other_senders_fullname_string = people
+        .get_display_full_names(displayed_other_senders.reverse())
```

Comment:
I should mention that, `reverse()` is highly inefficient[(Stackoverflow link)](https://stackoverflow.com/a/9113136). But we will always have 10 items in the list. So would it be a problem.

Reply:
Definitely not :).